src/Shared/FileMatcher.cs

```diff
@@ -2065,12 +2067,14 @@ private static string ComputeFileEnumerationCacheKey(string projectDirectoryUnes
             }
         }
 
-        enum SearchAction
+        public enum SearchAction
```

Comment:
Should the mode of `SearchAction` be modified, or should a different value be returned in the tuples?